### ISSUE TITLE
Feature-request: Is this possible to customize the Node as html elements

### ISSUE THREAD

**Issue by vikashpisces** [almende/vis#3798]
Wandering if this feature already exists, if not Can this be possible to have our own customized template node element on the diagram instead of having predefined node shape.

Basically, is it possible to connect html divs/template blocks using Vis.js?  This will help in having owr own customized node connected to each other.

**Comment by RanaProgrammer** [comment]
yeah i am also looking for this. @vikashpisces do you find a solution?

**Comment by vikashpisces** [comment]
@RanaProgrammer not yet as I need to see a custom developed plugin to create such html node unless we have something from Vis....

**Comment by kpcauchi** [comment]
I'm looking for this too. I think I will be forced not to use vis unfortunately.
If you find something let us know!

**Comment by vikashpisces** [comment]
@RanaProgrammer , @kpcauchi  - I found another library [jsPlumb ](https://jsplumbtoolkit.com/)(Community Edition which is free) and helps in connecting HTML elements like div, span etc to make  a network diagram.

**Comment by atopus** [comment]
I'm also interesting, for use with React component...

**Comment by kpcauchi** [comment]
I actually implemented a hack to pass a view.

I injected my view, used html2canvas to convert  my view to a canvas, then I got the data uri from the canvas and passed it as an image to the node.

**Comment by ventralnet** [comment]
Is this example what you guys are looking for?
http://visjs.org/examples/network/nodeStyles/HTMLInNodes.html

**Comment by kpcauchi** [comment]
Which browsers doesn't it work on? It works well on chrome it seems

**Comment by binDongKim** [comment]
Any news on this?
